### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

File: node/tests/k8st/infra/apiserver.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# This is a tech-preview manifest which installs the Calico API server. Note that this manfiest is liable to change
```
Comment (mgleung) [inline node/tests/k8st/infra/apiserver.yaml:1]: I think this is supposed to be `manifest`

File: node/tests/k8st/infra/apiserver.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+# This is a tech-preview manifest which installs the Calico API server. Note that this manliest is liable to change
```
Comment (mgleung) [inline node/tests/k8st/infra/apiserver.yaml:1]: I think this is supposed to be `manifest`

**Review by lwr20 — APPROVED** [review]

**Pull request merged**